render.c

```diff
@@ -34,7 +34,11 @@ void render(struct slurp_output *output) {
 		if (!seat->wl_pointer) {
```

Comment:
Unrelated to this PR, but: do we still need this check? It sounds like a touch-only device wouldn't render any selection because of this.

Reply:
Probably not. Unfortunately, I don't have a touch device that I can test this on.

Reply:
Opened #66 as a follow-up issue.